tachyon/crypto/sumcheck/multilinear/multilinear_sumcheck_unittest.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 arkworks contributors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.arkworks and the LICENCE-APACHE.arkworks
+// file.
+
+#include "tachyon/crypto/sumcheck/multilinear/multilinear_sumcheck.h"
+
+#include "gtest/gtest.h"
+
+#include "tachyon/base/range.h"
+#include "tachyon/math/finite_fields/test/finite_field_test.h"
+#include "tachyon/math/finite_fields/test/gf7.h"
+#include "tachyon/math/polynomials/multivariate/linear_combination.h"
+#include "tachyon/math/polynomials/multivariate/multilinear_dense_evaluations.h"
+
+namespace tachyon::crypto {
+namespace {
+
+using F = math::GF7;
+
+class MultilinearSumcheckTest : public math::FiniteFieldTest<F> {};
+
+}  // namespace
+
+TEST_F(MultilinearSumcheckTest, TestInteractiveProtocolSmall) {
+  // |SumcheckVerifier| runs in-depth logic in |InterpolateUniPoly()|
+  constexpr size_t num_variables = 2;
+  const base::Range<size_t> max_evaluations_in_term_range(2, 3);
+  constexpr size_t max_possible_evaluations = 3;
+  size_t num_terms = 2;
+  using MLE = math::MultilinearDenseEvaluations<F, num_variables>;
+  const math::LinearCombination<MLE> linear_combination =
+      math::LinearCombination<MLE>::Random(
+          num_variables, max_evaluations_in_term_range, num_terms);
+
+  CHECK(MultilinearSumcheck<MLE>::RunInteractiveProtocol<
+        max_possible_evaluations>(linear_combination, num_variables));
+}
+
+TEST_F(MultilinearSumcheckTest, TestInteractiveProtocolBig) {
+  // |SumcheckVerifier| returns early for |InterpolateUniPoly()|
+  constexpr size_t num_variables = 10;
+  const base::Range<size_t> max_evaluations_in_term_range(6, 30);
+  constexpr size_t max_possible_evaluations = 30;
```

Comment:
Please follow the constant naming convention! See https://google.github.io/styleguide/cppguide.html#Constant_Names